apps/src/applab/designProperties.jsx

```diff
@@ -48,6 +29,7 @@ var DesignProperties = module.exports = React.createClass({
     // TODO (brent) - right now if i create two elements with the same id, I
     // can still run into the same problem, where I click on the other element
     // and the table doesn't update
+    // TODO (brent) - it appears the wrong element sometimes gets deleted
```

Comment:
I saw it happen even when they didn't. Haven't spent the time to come up with a consistent repro yet.